service-worker.js

```diff
@@ -15,4 +15,8 @@ var preCachedRessources = [
 
 toolbox.precache(preCachedRessources);
 
-toolbox.router.get('/(.*)', toolbox.cacheFirst, swOptions);
+toolbox.router.get('/*', toolbox.cacheFirst, swOptions);
+toolbox.router.get('/*', toolbox.cacheFirst, { origin: "cdnjs.cloudflare.com" });
+toolbox.router.get('/*', toolbox.cacheFirst, { origin: "cdn.jsdelivr.net" });
+toolbox.router.get('/*', toolbox.cacheFirst, { origin: "unpkg.com" }); // for repl
+toolbox.router.post('/*', toolbox.cacheFirst, { origin: "algolia.net" }); // Cache Algolia search response
```

Comment:
@addyosmani I guess not. it should be networkFirst.

@xtuc `cacheFirst`, `networkFirst`, `fastest` - all these ultimately hit the cache in some way.